packages/react/src/utilities/hooks/useIsSmallScreen.ts

```diff
@@ -0,0 +1,16 @@
+import { useCallback, useSyncExternalStore } from "react";
+
+export const useIsSmallScreen = (breakPointToCheck: number) => {
+  const subscribe = useCallback((listener: () => void) => {
+    window.addEventListener("resize", listener);
+    return () => {
+      window.removeEventListener("resize", listener);
+    };
+  }, []);
+
+  const getSnapshot = useCallback(() => {
+    return window.innerWidth < breakPointToCheck;
+  }, [breakPointToCheck]);
+
+  return useSyncExternalStore(subscribe, getSnapshot);
```

Comment:
J'y pense juste un dernier changement pour ceux qui refait du rendu serveur

```suggestion
const getServerSnapshot = useCallback(() => false, []);
  
  return useSyncExternalStore(subscribe, getSnapshot, getServerSnapshot );
```